packages/@sanity/preview/src/components/SanityDefaultPreview.js

```diff
@@ -75,7 +75,7 @@ export default class SanityDefaultPreview extends React.PureComponent {
   renderIcon = options => {
     const {type} = this.props
     const Icon = type.icon
-    return Icon && <Icon />
+    return Icon && <Icon className="sanity-studio__preview-fallback-icon" />
```

Comment:
Why is this a hard coded string instead of an imported css modules class?

Reply:
Mainly because its need to be referred in the preview styling. Making it a css module would generate unreadable code and nested composed parts. 

This is easier to read
```
/* Mainly for styling fallback icons */
  @nest & :global(.sanity-studio__preview-fallback-icon) {
    width: 2em;
    height: 2em;
    opacity: 0.5;
  }
```

Reply:
I see 👍